nautobot_secrets_providers/providers/__init__.py

```diff
@@ -2,5 +2,10 @@
 
 from .aws import AWSSecretsManagerSecretsProvider
 from .hashicorp import HashiCorpVaultSecretsProvider
+from .thycotic import ThycoticSecretServerSecretsProvider
 
-__all__ = (AWSSecretsManagerSecretsProvider, HashiCorpVaultSecretsProvider)  # pylint: disable=invalid-all-object
+__all__ = (  # type: ignore
+    AWSSecretsManagerSecretsProvider,
+    HashiCorpVaultSecretsProvider,
+    ThycoticSecretServerSecretsProvider,
+)
```

Comment:
@glennmatthews Hi, I had to restore the __all__ as objects because in secrets.py the import will use this objects.

__all__ = (  # type: ignore
    AWSSecretsManagerSecretsProvider,
    HashiCorpVaultSecretsProvider,
    ThycoticSecretServerSecretsProvider,
)